src/components/select/index.tsx

```diff
@@ -0,0 +1,46 @@
+import clsx from 'clsx'
+import React from 'react'
+import { UseFormRegister, FieldValues } from "react-hook-form"
+
+import { SelectProps, SelectSize } from './types'
+
+export const Select = React.forwardRef<
+    HTMLSelectElement,
+    SelectProps & ReturnType<UseFormRegister<FieldValues>>
+>(({
+    error,
+    isDisabled,
+    label, 
+    name, 
+    onBlur,
+    onChange, 
+    options, 
+    size = SelectSize.Normal 
+}, ref) => (
+    <label className="form-control w-full max-w-xs">
+        {label && (
+            <div className="label">
+                <span className="label-text">{label}</span>
+            </div>
+        )}
+        <select 
+            className={clsx(`select select-bordered w-full max-w-xs ${size}`, {
```

Comment:
select select-bordered - kde je to definovaný? nebo to je z ty daisyUI?

Reply:
To je definovany v daisy ui